eq-author-api/schema/resolvers/withWritePermission.js

```diff
@@ -18,6 +19,10 @@ const withWritePermission = mutation => async (root, args, ctx) => {
   const result = await mutation(root, args, ctx);
   await saveQuestionnaire(ctx.questionnaire);
   ctx.validationErrorInfo = validateQuestionnaire(ctx.questionnaire);
+  pubsub.publish("validationUpdated", {
```

Comment:
Could we push this down into save questionnaire, seems pretty unrelated to WritePermission and I don't think we ever want to call save and not validate?

Reply:
Now renamed to `createMutation` so this wrapper no longer exists